python_modules/dagster/dagster/_core/events/__init__.py

```diff
@@ -1763,6 +1790,19 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+class RunEnqueuedData(
+    NamedTuple(
+        "_RunEnqueuedData",
+        [
+            ("code_location_name", str),
+            ("repository_name", str),
+        ],
+    )
+):
+    pass
```

Comment:
only thing that gives me pause is if there are any back-compat challenges here with users loading existing events over the API? I think its likely fine since its a brand new field, just need to think it through for a sec

Reply:
we won't actually send these to them 

https://github.com/dagster-io/dagster/blob/master/python_modules/dagster-graphql/dagster_graphql/implementation/events.py#L326-L330

Reply:
also think it should be fine 

nit: can use `@record`